src/java.desktop/macosx/native/libawt_lwawt/awt/a11y/TabButtonAccessibility.m

```diff
@@ -59,11 +59,14 @@ - (void)dealloc
 - (jobject)tabGroup
 {
     if (fTabGroupAxContext == NULL) {
-        JNIEnv* env = [ThreadUtilities getJNIEnv];
-        jobject tabGroupAxContext = [(CommonComponentAccessibility *)[self parent] axContextWithEnv:env];
-        fTabGroupAxContext = (*env)->NewWeakGlobalRef(env, tabGroupAxContext);
-        CHECK_EXCEPTION();
-        (*env)->DeleteLocalRef(env, tabGroupAxContext);
+        id parent = [self parent];
```

Comment:
Probably we can sanitize all casts to (CommonComponentAccessibility *) via some utility method which will check the type and then cast, so all calling code will get a CommonComponentAccessibility or nil?

Reply:
@mrserb Thanks, I did that.  I had to re-submit PR for jdk18, please proceed in https://github.com/openjdk/jdk18/pull/65